std/uni.d

```diff
@@ -7241,7 +7241,8 @@ private static struct InputRangeString
     $(P Useful for converting the result to a string after doing operations
     on graphemes.)
 
-    $(P Acts as the identity function when given a range of code points.)
+    $(P If passed in a range of codepoint will return an opaque range of code points
```

Comment:
"If passed in a range of code points, will return ..."

Reply:
Will do